docroot/modules/custom/va_gov_vamc/src/Plugin/Validation/Constraint/SectionMatcherValidator.php

```diff
@@ -19,16 +19,19 @@ public function validate($items, Constraint $constraint) {
     $fieldLabel = $items->getFieldDefinition()->getLabel();
     $nodeStorage = \Drupal::entityTypeManager()->getStorage('node');
     $termStorage = \Drupal::entityTypeManager()->getStorage('taxonomy_term');
-    $sectionName = $termStorage->load($sectionTermID)->getName();
-    foreach ($items as $item) {
-      $referencedEntity = $nodeStorage->load($item->target_id);
-      if ($referencedEntity->field_administration->target_id !== $sectionTermID) {
-        /** @var \Drupal\va_gov_vamc\Plugin\Validation\Constraint\SectionMatcher $constraint */
-        $this->context->addViolation($constraint->notSectionMatch, [
-          '%section' => $sectionName,
-          '%fieldLabel' => $fieldLabel,
-        ]);
-        return;
+    // If the node is new, no Section has been set; so do not validate.
+    if (isset($sectionTermID)) {
```

Comment:
This seems like an odd check.  The code itself sets this. 
![image](https://user-images.githubusercontent.com/5752113/226799508-a04bb479-4afd-4d0b-964e-5f2d143267c7.png)

So what is actually being checked is whether $sectionTermID is not NULL.  Which would fail the isset.  It is more explicit (better readability) to look for what you are actually trying to pivot on.  
```suggestion
    if (!empty($sectionTermID)) {
```